indico_zodbimport/modules/event_layout.py

```diff
@@ -83,12 +86,23 @@ def migrate_layout_settings(self):
                                      event_id=event.id)
                     continue
                 with open(path, 'rb') as f:
-                    logo_content = f.read()
-                logo_content_type = mimetypes.guess_type(logo.fileName)[0]
+                    try:
+                        logo_image = Image.open(f)
+                        logo_content_type = 'image/png'
+                        logo_bytes = BytesIO()
+                        logo_image.save(logo_bytes, 'PNG')
+                        logo_bytes.seek(0)
+                        logo_content = logo_bytes.read()
+                        logo_filename = (os.path.splitext(secure_filename(convert_to_unicode(logo.fileName),
+                                                                          'logo'))[0] + '.png')
+                    except IOError:
+                        logo_content = f.read()
```

Comment:
That was my first approach. However I think we should leave it to the users to change it.